rust-src/concordium_base/src/curve_arithmetic/mod.rs

```diff
@@ -212,46 +212,70 @@ pub fn multiexp_worker<C: Curve, X: Borrow<C>>(
 pub fn multiexp_worker_given_table<C: Curve>(
     exps: &[C::Scalar],
     table: &[Vec<C>],
-    window_size: usize,
+    window: usize,
 ) -> C {
     // Compute the wnaf
-
+    let window_size = window + 1;
     let k = exps.len();
     // assert_eq!(gs.len(), k);
-    assert!(window_size >= 1);
     assert!(window_size < 62);
```

Comment:
I think for consistency with the rest of the code, this should also be `assert!(window_size < 63);`.

Reply:
Ok.